module_bindings/dripline_core/_endpoint.hh

```diff
@@ -21,21 +21,28 @@ namespace dripline_pybind
         pybind11::class_< dripline::endpoint, endpoint_trampoline, std::shared_ptr< dripline::endpoint > >( mod, "_Endpoint", "Endpoint binding" )
                 .def( pybind11::init< const std::string& >(),
                         pybind11::call_guard<pybind11::scoped_ostream_redirect,
-                                       pybind11::scoped_estream_redirect>() )
+                                pybind11::scoped_estream_redirect>() )
                 .def( "submit_request_message", &dripline::endpoint::submit_request_message,
-                        pybind11::call_guard<pybind11::scoped_ostream_redirect,
-                                       pybind11::scoped_estream_redirect>() )
+                        pybind11::call_guard<pybind11::gil_scoped_release,
```

Comment:
Didn't we determine that the `gil_scoped_release` has to go after the `scoped_o(e)stream_redirect`s?